charts/capsule/templates/deployment.yaml

```diff
@@ -81,7 +81,7 @@ spec:
             allowPrivilegeEscalation: false
         - name: kube-rbac-proxy
           image: {{ .Values.proxy.image.repository }}:{{ .Values.proxy.image.tag }}
-          imagePullPolicy: IfNotPresent
+          imagePullPolicy: {{ .Values.proxy.image.pullPolicy }}
```

Comment:
What about using a template function as we did with the manager container?

https://github.com/clastix/capsule/blob/4f34483dee32bcd8ff88c7590f1ab8e5cdcdfa9e/charts/capsule/templates/_helpers.tpl#L64:L69